vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -995,6 +995,72 @@ The absence of `RestrictedProperties` denotes all properties are accessible usin
         <Annotation Term="Core.Description" String="Stream property supports update of its media edit URL and/or media read URL" />
       </Term>
 
+      <Term Name="DefaultCapabilities" Type="Capabilities.DefaultCapabilitiesType" Nullable="false" AppliesTo="EntityContainer">
+        <Annotation Term="Core.Description" String="Default capability settings for all collection-valued resources in the container. Annotating a capability term for a specific collection-valued resource overrides the default setting" />
+      </Term>
+      <ComplexType Name="DefaultCapabilitiesType">
+        <Property Name="ChangeTrackingSupported" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="`odata.track-changes` preference is supported" />
+        </Property>
+        <Property Name="IndexableByKey" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Supports key values according to OData URL conventions" />
+          <Annotation Term="Core.LongDescription" String="Can be overruled by annotating term `IndexableByKey` on an entity set or term `NavigationRestrictions`, property `RestrictedProperties/IndexableByKey` for a navigation property path starting at a set" />
+        </Property>
+        <Property Name="CountSupported" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Entities can be counted (only valid if targeting an entity set)" />
+          <Annotation Term="Core.LongDescription" String="Can be overruled by annotating term `CountRestrictions` on an entity set" />
+        </Property>
+        <Property Name="TopSupported" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="`$top` is supported" />
+          <Annotation Term="Core.LongDescription" String="Can be overruled by annotating term `TopSupported` on an entity set or term `NavigationRestrictions`, property `RestrictedProperties/TopSupported` for a navigation property path starting at a set" />

```

Comment:
```suggestion
          <Annotation Term="Core.LongDescription" String="Can be overruled by annotating term `TopSupported` on a collection or term `NavigationRestrictions`, property `RestrictedProperties/TopSupported` for a navigation property path starting at a set" />
```
if you want to use the individual terms for collection-valued _structural_ properties.